variables.tf

```diff
@@ -117,6 +117,12 @@ variable "deploy_db" {
   description = "Do you want a database deployed in this project?"
 }
 
+variable "db_version" {
+  type        = string
+  default     = "POSTGRES_9_6"
```

Comment:
Can we add a comment here with a hyperlink to GCP docs listing available versions? That would be really helpful to anybody who is using this module.

Reply:
```suggestion
  # https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_database_instance#argument-reference
  default     = "POSTGRES_9_6"
```